examples/typescript.ts

```diff
@@ -1,3 +1,9 @@
+let someVariable = "Hello World!";
+const someConstant = "Hello World!";
+
+console.log(someConstant);
+console.log(someVariable);
+
```

Comment:
Suggestion from what I noticed in another screenshot earlier, highlight `@variable.builtin` distinctly (console, arguments, etc)

Reply:
Interested to get your thoughts here because `@variable.typescript` takes priority over `@variable.builtin.typescript`.

<img width="547" alt="Screen Shot 2023-03-04 at 09 45 48@2x" src="https://user-images.githubusercontent.com/9512444/222892103-48cea53c-47d6-46da-8b5c-5b12029d9050.png">

Reply:
That's bizarre, it makes me think the issue lies in the theme and not nvim-treesitter's queries...

This is the actual queries file (builtin is below standard variables)
![image](https://user-images.githubusercontent.com/29718261/222892368-87b76542-58c1-46bf-a8ed-fb94edea85ff.png)